ui/src/router/components/form/components/router_config/GeneralSettingsPanel.js

```diff
@@ -52,6 +52,7 @@ export const GeneralSettingsPanel = ({
             value === "UPI_V1"
               ? "/caraml.upi.v1.UniversalPredictionService/PredictValues"
               : "",
+          endpoint: "",
```

Comment:
reset endpoint as well when protocol cause, since the endpoints are not re-usable for both protocol